docs/getting-started/supported-languages-and-tools.md

```diff
@@ -50,9 +50,14 @@ The table below lists all languages that Codacy supports and the corresponding t
   <tbody>
     <tr>
       <td>Apex</td>
+      <!--
       <td><a href="https://pmd.github.io/">PMD</a>,
           <a href="https://semgrep.dev/">Semgrep</a> <a href="#semgrep"><sup>1</sup></a></td>
       <td>-</td>
+      <td><a href="https://semgrep.dev/">Semgrep</a></td>
```

Comment:
to uncomment this we need to first add Language.Apex as a supported language to Semgrep in codacy/codacy-tools